### ISSUE TITLE
Discovery | Design | Profile | Research Recommendations | Address Validation

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#48537]
## Background

During the recent Accessibility Audit as well as the research sessions exploring the "One Thing Per Page" concept, the address validation and editing flows were noted as being difficult to navigate for both screenreader users as well as non screenreader users.

Additionally, during our BAI mid-point review we received feedback on the 'Edit Address' and 'edit it' links (shown when the US Postal service doesn't have a matching address) as essentially buttons styled to look like links, which is not a practice encouraged ("buttons look like buttons and do actions, links look like links and go places"). More details on why this can be problematic can be found in #43095.

## Tasks
- [x] Explore options to improve visibility and understanding for the “confirm your address”  message
- [x] Explore improving visibility of edit buttons to help users understand how to edit their info
- [x] Look for alternative options  to the "buttons styled as links" 
- [x] Create mocks / flows depicting recommended options

## Acceptance Criteria
- [x] Share solutions with Profile team and align on best options to pursue for either further exploration or implementation
- [x] Notify PM when ready for next steps

## Associated Tickets
- #43095

**Comment by SKasimow** [comment]
Hi @andaleliz - can you review this ticket and lmk if anything needs to be changed/added before I pull into next grooming meeting? Thanks!

cc @ericasauv09

**Comment by andaleliz** [comment]
Adding the ticket from the first time we visited this for reference: https://github.com/department-of-veterans-affairs/va.gov-team/issues/30258

And the [FE documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/address-validation/designs/2021-alert-updates/README.md) I wrote up about the UX:

**Comment by andaleliz** [comment]
I've explored a few things, and landed on 2 ideas. I'm waiting to hear back from Danielle on the exact wording:

1. To solve the problem of buttons that look like links, we can update a button label. Instead of having the cancel button label read "cancel", we can update it to "Go back to editing" so it's clear to users that they have a way back, and we can get rid of the buttons that look like links elsewhere in the UI.  We can also clean up the alert content. This would be a relatively easy and quick thing we can do.

![image](https://user-images.githubusercontent.com/55992116/200347143-e87f1af7-8804-49bb-8cea-3bcbafe30e3d.png)

2.  This doesn't fully solve the problem of people not realizing they've triggered address validation and moving onto something else.  To solve that, I re-visited the modal solution we came up with last year. It really doesn't work when combined w/ the address change modal - there's too much content and I'm confident people won't read it all. Here's a video where you can see that UX:

https://user-images.githubusercontent.com/55992116/200348270-e2b56c34-bf1e-4ee6-82a7-8a66a233e817.mov

I think this is is another kind of sub-task where "one thing per page" might work - I put together [a quick prototype](https://preview.uxpin.com/72a4060dec9250e4385a9abfaeefec42b3994e00#/pages/159609186/simulate/sitemap?mode=i) to show how it could go.  If you click `save` on the first screen, you can go through the flow.  

I asked some high level questions of FE and a11y and think this is a viable option.  

@Samara-Strauss, what are your thoughts?

**Comment by andaleliz** [comment]
>I'm unsure if that'll feel natural or weird to go from in-line editing to a new page for this particular part of the flow

**Comment by andaleliz** [comment]
We've discussed this with the team, and in light of the current contract status, we're going to hold off on further work for the "one thing per page" solution, and implement option 1. That is ready for FE, and the mock-ups are all available from the [use case documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/use-cases/address-validation.md) (6 in total).

The updates will involve:
- updating the secondary button label text
- updating the secondary button so it returns the user to edit mode instead of read mode
- updating the last sentence in all but 1 alert

I noticed that on staging, when the address can't be confirmed, and a suggestion is **not** found, we're showing the alert for when a suggestion is found, so FE will need to double check the logic on that too.

I updated our [use case documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/use-cases/address-validation.md)  with a "Note for future iterations" capturing other things that have been discussed in this ticket.
